doc/design/proposal-chaining-iterative.md

```diff
@@ -282,3 +282,53 @@ const babelOutputToFormat = new Map([
 ]);
 ```
 </details>
+
+## Chaining `readFile` hooks
+
+Say you had a chain of three loaders:
+
+* `zip` adds a virtual filesystem layer for in-zip access
+* `tgz` does the same but for tgz archives
+* `warc` does the same for warc archives.
```

Comment:
These are all types of archives. Would this hook be leveraged only be archive loaders? Not saying that's not enough, but if so, I think there should be a note to that effect. If it's not so, then I think one of them (maybe warc) could be replaced with something else that is not an archive.

Reply:
I replaced `warc` by `https`, since loading through the network is another example